horovod/spark/lightning/remote.py

```diff
@@ -316,9 +327,9 @@ def calculate_shuffle_buffer_size():
 
 def _create_dataloader(feature_columns, input_shapes, metadata, data_loader_cls=None):
     if data_loader_cls is None:
-        # set PytorchAsyncDataLoader as default
-        from horovod.spark.data_loaders.pytorch_data_loaders import PytorchAsyncDataLoader
-        data_loader_cls = PytorchAsyncDataLoader
+        # set PytorchInfiniteAsyncDataLoader as default
+        from horovod.spark.data_loaders.pytorch_data_loaders import PytorchInfiniteAsyncDataLoader
```

Comment:
What's the reason of using this `PytorchInfiniteAsyncDataLoader`, instead of setting epochs = None?

Reply:
We do not need to create new data loader in each epoch. 
This PytorchInfiniteAsyncDataLoader  requires epochs == None in reader.